docker/staging/exporter-conf/config.yaml

```diff
@@ -0,0 +1,6 @@
+default-group:
+  location: default
+  NetworkSerialPort:
```

Comment:
Could you use a USBSerialPort instead? labgrid-exporter would then manage the ser2net process itself.